src/lib.rs

```diff
@@ -110,3 +110,13 @@ pub type Radians<T> = Length<T, Rad>;
 
 /// A value in Degrees.
 pub type Degrees<T> = Length<T, Deg>;
+
+/// Temporary alias to facilitate the transition to the new naming scheme
+pub type Matrix2D<T> = Transform2D<T>;
```

Comment:
Perhaps, we should put `#[deprecated]` on top? See https://github.com/rust-lang/rust/issues/29935